specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2024-05-01-preview/appplatform.json

```diff
@@ -8996,6 +9000,32 @@
         }
       }
     },
+    "ServiceManagedIdentityProperties": {
```

Comment:
I happen to came across this file `https://github.com/Azure/azure-rest-api-specs/blob/4f093ebabe4525a5d10ca57e5804523a27dce7bf/specification/common-types/resource-management/v3/managedidentity.json#L76C1-L110C8`

I think you can refer to it.

Reply:
Yes, we can just refer that one. Updated, pls check